core-services/prow/02_config/_plugins.yaml

```diff
@@ -322,14 +322,10 @@ bugzilla:
       target_release: 4.4.z
       validate_by_default: true
     openshift-4.5:
-      dependent_bug_states:
```

Comment:
Should we land this now? QE should be testing 4.6, right?

Reply:
They are, I'm just verifying or managing that by exception on my own right now.